### DISCUSSION TITLE
Im asking for help ? This is a first yet it is for tbe bettermwnt..

### DISCUSSION THREAD

**Discussion by Jennyother7** [community/community#121874] [Other Feature Feedback, Questions, & Ideas]
Can we use voice? I cant see screen

**Comment by Kumdoreda** [comment]
Hello, if I sent funds using Scroll Bridge but did not receive the assets, where can I contact them?

**Comment by ebndev** [comment]
Hi @Jennyother7,

Welcome to the Community! Can you clarify what you need help with?
